packages/orchestrator-ui-components/src/components/WfoSubscription/WfoInSyncField.tsx

```diff
@@ -30,20 +33,21 @@ export const WfoInSyncField = ({ subscriptionDetail }: WfoInSyncFieldProps) => {
         subscriptionDetail.processes.page,
     );
     const [setSubscriptionInSync] = useSetSubscriptionInSyncMutation();
+    const { isFetching } = useGetSubscriptionDetailQuery({
```

Comment:
This is fine aswell but I would pass the prop down from the parent component also calling useGetSubS...